commands/faas.go

```diff
@@ -33,6 +33,7 @@ var (
 	handler      string
 	image        string
 	language     string
+	debug        bool
```

Comment:
the `debug` flag is a design change.. can you explain what it does? Should this just be "--verbose" perhaps for the `template pull` command?

Reply:
sure I can switch it to a flag just for the pull command